employee/build.gradle.kts

```
@@ -32,6 +32,7 @@ dependencies {
 	annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 	testImplementation("org.springframework.boot:spring-boot-starter-test")
 	implementation("com.mysql:mysql-connector-j:8.0.33")
+	compileOnly("org.flywaydb:flyway-core:9.19.1")
```

```suggestion
	implementation("org.flywaydb:flyway-core:9.19.1")
```
implementation 써야하지않나요?

---

아니네요 plugin으로 넣어야하네요
https://flywaydb.org/documentation/usage/gradle/